xblockutils/public/studio_container.js

```diff
@@ -1,20 +1,21 @@
 function StudioContainerXBlockWithNestedXBlocksMixin(runtime, element) {
     var $buttons = $(".add-xblock-component-button", element),
+        $view = $('.add-xblock-component', element),
```

Comment:
@itsjeyd I named it `view` because it points to the same element that edx-platform sets up for the Backbone view. Do you think `component` is better?